ironic/manifest.jsonnet

```diff
@@ -0,0 +1,249 @@
+local kpm = import "kpm.libjsonnet";
+
+function(
+  params={}
+)
+
+kpm.package({
+  package: {
+    name: "stackanetes/ironic",
+    expander: "jinja2",
+    author: "Mateusz Blaszkowski",
+    version: "0.1.0",
+    description: "ironic",
+    license: "Apache 2.0",
+  },
+
+  variables: {
+    deployment: {
+      control_node_label: "openstack-control-plane",
+      compute_node_label: "openstack-ironic-compute-node",
+      replicas: 1,
+
+      image: {
+        base: "quay.io/stackanetes/stackanetes-%s:barcelona",
+
+        init: $.variables.deployment.image.base % "kolla-toolbox",
+        post: $.variables.deployment.image.base % "kolla-toolbox",
+        # TODO:
+        api: "10.91.96.87:5000/debian-source-ironic-api:mateuszb-ironic",
+        db_sync: "10.91.96.87:5000/debian-source-ironic-api:mateuszb-ironic",
+        conductor: "10.91.96.87:5000/debian-source-ironic-conductor:mateuszb-ironic",
+        compute: "10.91.96.87:5000/debian-source-nova-compute-ironic:mateuszb-ironic",
+        ironic_pxe: "10.91.96.87:5000/debian-source-ironic-pxe:mateuszb-ironic",
+        ironic_http: "nginx:alpine",
```

Comment:
Agreed to leave public nginx image - i'll just use more precise (nginx:1.11.5-alpine) version